analyzer/src/steps/typing.rs

```diff
@@ -777,6 +777,64 @@ fn ascribe_substitution(
     }
 }
 
+fn ascribe_detached(
+    detached: &Detached,
+    exploration: &mut Exploration,
+    links: Links,
+    diagnostics: &mut Vec<Diagnostic>,
+    state: TypingState,
+) -> TypedExpr {
+    let expr = ascribe_types(
+        exploration,
+        links,
+        diagnostics,
+        &detached.underlying,
+        state.with_local_value(ExpressionValue::Unused),
+    );
+    TypedExpr {
+        kind: ExprKind::Subprocess(Subprocess {
+            inner: Box::new(expr),
+            awaited: false,
+        }),
+        ty: INT,
```

Comment:
why not introduce a PID type ?

Reply:
I have mixed feelings about this. On one hand, this would be useful to avoid misuses and on the other hand, the current native system isn't ready to nicely integrate type aliases/native wrapper types.